packages/backend/src/app.ts

```diff
@@ -138,14 +138,12 @@ export interface AppServices {
 export type AppContainer = IocContract<AppServices>
 
 export class App {
-  private koa!: Koa<DefaultState, AppContext>
-  private publicRouter!: Router<DefaultState, AppContext>
+  private openPaymentsServer!: Server
   private server!: Server
```

Comment:
nit: maybe we can call this `adminServer` instead of just `server` so the names are clear on what each of the servers on the app are.